packages/content/types/parsers/markdown/index.d.ts

```diff
@@ -0,0 +1,24 @@
+export interface IMarkdown {
```

Comment:
options can be typed

Reply:
should be class

Reply:
Updated it to a class with correct options according to documenation